index.html

```diff
@@ -28,6 +28,9 @@
         <link rel="mask-icon" href="https://swift.org/assets/images/icon-swift.svg" color="#F05339" />
         <title>Swift Evolution</title>
         <link href="index.css" rel="stylesheet" type="text/css">
+        <meta name="og:title" content="Swift Evolution" />
+        <meta name="og:description" content="This maintains proposals for changes and user-visible enhancements to the Swift Programming Language." />
```

Comment:
```suggestion
        <meta name="og:description" content="Proposals for changes and user-visible enhancements to the Swift Programming Language." />
```

Just a suggestion for the wording. Not sure if you have a screenshot of how this would look on social media?

Reply:
Sorry I don't have any screen shot at the moment but we can test it in following site -: https://metatags.io/